code/__HELPERS/_string_lists.dm

```diff
@@ -12,11 +12,11 @@ var/global/list/string_cache
 			var/list/stringsList = list()
 			fileList = file2list("strings/[filename]")
 			for(var/s in fileList)
-				stringsList = text2list(s, "@=")
+				stringsList = splittext(s, "@=")

```

Comment:
So, i'm just gonna point out, that this doesn't actually work.

luckly it will most likely work in 510.1322

http://www.byond.com/forum/?post=2026759&page=2